src/analyzer/protocol/irc/IRC.h

```diff
@@ -75,6 +75,14 @@ class IRC_Analyzer final : public analyzer::tcp::TCP_ApplicationAnalyzer {
      */
     std::vector<std::string> SplitWords(const std::string& input, char split);
 
+    /**
+     * Checks if a passed string is a valid command for an IRC client.
+     *
+     * \param command command to check
+     * \return true if command is valid
+     */
+    static bool IsValidClientCommand(const std::string& command);
```

Comment:
No strong opinion and the C++ gurus might disagree, but if it's private static, maybe can just make it a static/anonymous namespace function in `IRC.cc`?

Reply:
Yup, I could :)

This is purely for visibility, and there is no drawback that I am aware of to do it this way.